pkg/cli/main_test.go

```diff
@@ -31,10 +30,6 @@ func TestMain(m *testing.M) {
 	// a version injected. Pretend to be a very up-to-date version.
 	defer build.TestingOverrideTag("v999.0.0")()
 
-	// The times for Example_statement_diag are reported in the local timezone.
```

Comment:
Why did you change this?

Reply:
I didn't plan to check-in the deletion, I just wanted to see if that was the cause of the`make stress` failures for the dump unit tests on CI. I have asked Radu for his input on the same in the comment above - https://github.com/cockroachdb/cockroach/pull/51185#pullrequestreview-446066146